### PR TITLE
Resourceful controllers and routes, Eloquent Models and relations amo…

### PR DESCRIPTION
**Pull request by Akhwandzada1**
…ng them.

### CODE REVIEW

File: app/Http/Controllers/EmployeeController.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class EmployeeController extends Controller
+{
+    //
```
Comment (zeeshanmughaltherightsw) [inline app/Http/Controllers/EmployeeController.php]: Make this controller a resource controller

File: app/Http/Controllers/ProjectController.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class ProjectController extends Controller
+{
+    //
```
Comment (zeeshanmughaltherightsw) [inline app/Http/Controllers/ProjectController.php]: Make this controller as resource as well

File: database/migrations/2023_06_20_131119_create_project_employee_table.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('project_employee', function (Blueprint $table) {
```
Comment (zeeshanmughaltherightsw) [inline database/migrations/2023_06_20_131119_create_project_employee_table.php]: be carefull when you are naming the table

File: database/migrations/2023_06_20_131119_create_project_employee_table.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            $table->id();
```
Comment (zeeshanmughaltherightsw) [inline database/migrations/2023_06_20_131119_create_project_employee_table.php]: when you create pivot table dont add Id and timestamps columns

**Review by zeeshanmughaltherightsw — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/EmployeeController.php (hunk 1 above)
Comment (Akhwandzada1) [reply app/Http/Controllers/EmployeeController.php]: Ok

File: app/Http/Controllers/ProjectController.php (hunk 2 above)
Comment (Akhwandzada1) [reply app/Http/Controllers/ProjectController.php]: Ok

File: database/migrations/2023_06_20_131119_create_project_employee_table.php (hunk 3 above)
Comment (Akhwandzada1) [reply database/migrations/2023_06_20_131119_create_project_employee_table.php]: Ok

File: database/migrations/2023_06_20_131119_create_project_employee_table.php (hunk 4 above)
Comment (Akhwandzada1) [reply database/migrations/2023_06_20_131119_create_project_employee_table.php]: Ok

**Pull request merged**